Documentation/PackedAssetBuilds.md

```diff
@@ -1,12 +1,40 @@
 # Packaged MultiplayerSample Builds
+
 To make relocatable client and server builds for the MultiplayerSample, we recommend making packaged builds. These package builds will contain the Game or Server Launcher and the bundled assets needed to run the launcher outside of the developer environment.
 
 You can make both release packaged builds or profile packaged builds. For more information about creating release builds, see the O3DE documentation on [Creating a Project Game Release Layout for Windows](https://www.o3de.org/docs/user-guide/packaging/windows-release-builds/).
 
 The guide below covers how to make profile packaged builds which are very useful for early sharing and play testing.
 
+## Brief outline of the packaging steps
+
+1. Install WWise SDK (required for o3de-multiplayersample but optional for other projects)
+1. Use the engine from GitHub, not the installer
+1. Create a build for non-monolithic build of the engine
+1. Compile the engine with o3de-multiplayersample in profile non-monolithic mode
+1. Process all the assets using the Asset Processor from profile non-monolithic mode
+1. Create game bundles (.pak files) from the seed lists (a list of game assets)
+1. Create a build for monolithic build of the engine
+1. Compile the engine with o3de-multiplayersample in profile (and/or release) monolithic mode
+1. Create a folder for the packaged build outside of the engine, this will be the share-able folder of the packaged game
+1. Copy the monolithic game binaries and the game bundles to the packaged folder
+1. The packaged build is ready and can be shared without requiring the developer environment.
+
+## Pre-requisites
+
+*Important*: you can not use an installer to produce packaged builds. In order to produce a packaged build you will need to build the engine from source from GitHub.
+
+Additionally, we will need to compile the engine in non-monolithic profile build and in monolithic profile (and optionally release) mode.
+
+> Note, a monolithic build is a special build of O3DE engine and projects that combines all the gems used by the project into a single game executable. Additionally, monolithic builds do not build the Editor or the Asset Processor. Thus, a regular non-monolithic build is used to process assets and bundle them into .pak files, while a monolithic build is used to produce share-able game binaries.
+
+This guide uses `C:\git\o3de` is the source for the O3DE, cloned from GitHub.
+`C:\git\o3de-multiplayersample` is the location of the clone project. These paths are optional. If you choose to use different paths, amend the command scripts in the rest of the guide accordingly.
+
 ## Windows Profile PAK Setup
 
+Multiplayer Sample uses WWise gem and assets for audio effects. O3DE engine (and O3DE installers) do not include WWise support by default. In order to add the WWise support in the engine, one must first install WWise SDK and then re-build O3DE engine from source.
```

Comment:
I'm pretty sure the proper spelling is `Wwise` and not `WWise`  https://www.audiokinetic.com/en/products/wwise/
```suggestion
Multiplayer Sample uses the Wwise gem for audio effects and so we must first install the Wwise SDK.
```